SwiftLeeds/Extension/Color+Extension.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Color+Extension.swift
+//  SwiftLeeds
+//
+//  Created by Matthew Gallagher on 14/11/2021.
+//
+
+import SwiftUI
+
+extension Color {
+    static let background = Color("Background")
```

Comment:
Super nitpicky, but using .background as a variable name in SwiftUI leads to some really buggy autocomplete as it conflicts with the `.background()` modifier. Think it's worth changing this early days to `brandOrange` or even better, setting it as the accent color in the asset catalogue ( so you then get it as default tint for free, and can use .accent ) instead?

Reply:
This was a temporary option as was hoping to speak about assent naming when we got together.  The accent colour is more the tint, but can set to accent for now.  I wouldn't want to use a colour in the name, I'd prefer something more like primary or more descriptive

Reply:
Sounds like it'll be handled later, nice :)